package.json

```diff
@@ -22,6 +37,77 @@
   },
   "homepage": "https://github.com/freeCodeCamp/chapter#readme",
   "dependencies": {
+    "babel-plugin-module-resolver": "^3.2.0",
+    "express": "^4.17.1",
+    "express-async-handler": "^1.1.4",
+    "express-response-errors": "^1.0.4",
+    "fork-ts-checker-webpack-plugin": "^1.5.1",
+    "module-alias": "^2.2.2",
+    "morgan": "^1.9.1",
+    "next": "^9.1.1",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
+    "ts-node": "^8.4.1",
+    "typescript": "^3.6.4"
+  },
+  "devDependencies": {
+    "@testing-library/react": "^9.3.0",
+    "@types/express": "^4.17.1",
+    "@types/jest": "^24.0.19",
+    "@types/morgan": "^1.7.37",
+    "@types/next": "^8.0.6",
+    "@types/node": "^12.11.1",
+    "@types/react": "^16.9.9",
+    "@types/react-dom": "^16.9.2",
+    "@typescript-eslint/eslint-plugin": "^2.4.0",
+    "@typescript-eslint/parser": "^2.4.0",
+    "eslint": "^6.5.1",
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-config-react": "^1.1.7",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "husky": "^3.0.9",
+    "jest": "^24.9.0",
+    "lint-staged": "^9.4.2",
+    "node-typescript": "^0.1.3",
+    "nodemon": "^1.19.4",
+    "prettier": "^1.18.2",
     "speccy": "^0.11.0"
+  },
+  "lint-staged": {
+    "*.{ts,tsx}": [
+      "npm run pretty",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "jest": {
+    "watchPathIgnorePatterns": [
+      "/node_modules"
+    ],
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js"
```

Comment:
Just curious - do we not need a `"jsx"` here?

Reply:
We should have it for consistency - I'll add it - thanks!